src/controllers/cohortController.ts

```diff
@@ -98,3 +120,29 @@ export const getMyApplicationController = async (
     return next(error)
   }
 }
+
+export const getCohortOverviewController = async (
```

Comment:
we shouldn't name individual function as controller. The whole file is a controller but individual functions are just functions of endpoints